content/04.results.md

```diff
@@ -41,3 +41,28 @@ We observed that including one label function of a mismatched type to distant su
 We found that usually adding random noise did not improve performance (Figure {@fig:auroc_random_label_function_performance} and Supplemental Figure {@fig:aupr_random_label_function_performance}).
 For the CbG edge type we did observe slightly increased performance via AUPR (Supplemental Figure {@fig:aupr_random_label_function_performance}).
 However, performance changes in general were smaller than those observed with mismatched label types.
+
+### Discriminative Model Performance
+
+![
+The discriminator model improves together with the generative model as more edge-specific label function are included.
+The line plot headers represents the specific edge type the discriminator model is trying to predict.
+The x-axis shows the number of randomly sampled label functions that are incorporated on top of the baseline model (point at 0).
+The y axis shows the area under the receiver operating curve (AUROC).
+Each datapoint represents the average of each 50 sample run and the error bars represent the 95% confidence interval of each run.
+The baseline and “All” data points consist of sampling from the entire fixed set of label functions.
+This makes the error bars appear flat.
+](https://raw.githubusercontent.com/danich1/snorkeling/1941485a02c8aa9972c67d8f9d3ff96acb0f3b7b/figures/disc_model_experiment/disc_model_test_auroc.png){#fig:auroc_discriminative_model_performance}
+
+The discriminator model is designed to augment performance over the generative model by incorporating textual features along with estimated training labels.
+The discriminative model is a piecewise convolutional neural network trained over word embeddings (See Methods).
+We report the results of the discriminative model using AUROC (Figure {@fig:auroc_discriminative_model_performance} and AUPR (Supplemental Figure {@fig:aupr_discriminative_model_performance}).  
+  
+We found that the discriminative model generally out-performed the generative model as more edge-specific label functions are incorporated.
+The discriminator model's performance is modest when less than eleven edge-specific label functions are added to the baseline model (seen in Disease associates Gene (DaG), Compound binds Gene (CbG) and Gene interacts Gene (GiG)). 
+This suggests that a lower bound of at least eleven label functions is needed to generate training labels the discriminator model can effectively utilize.
+An exception to this trend is Compound treats Disease (CtD) where the discriminator model out-performs the generative model at all levels of sampling.
+Regarding CbG the discriminator model performs subpar with the generative model until all edge-specific label functions are used.
+Interestingly, the AUPR for CbG plateaus below the generative model and the decreases when all edge-specific label functions are used (Supplemental Figure {@fig:aupr_discriminative_model_performance}).
+This suggests that the discriminator model might be predicting more false positives than we expected.
```

Comment:
```suggestion
This suggests that the discriminator model might be predicting more false positives in this setting.
```
I think this is what you're going for. Otherwise I'm not sure where the expectation comes from. I may be confused.